### PR TITLE
Basic Salary Calculator

### PR DESCRIPTION
**Pull request by black-sonic4**
This is just a random sample 
Ok great

### CODE REVIEW

File: EmployeesSalary/Program.cs [hunk 1]
```diff
@@ -1,2 +1,52 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
```
Comment (black-sonic4) [inline EmployeesSalary/Program.cs:2]: Why we did remove this?

Comment (RamaRehawe) [reply EmployeesSalary/Program.cs:2]: unneeded

**Comment by black-sonic4** [comment]
Looks Good

**Comment by RamaRehawe** [comment]
Good!!

**Pull request merged**